parlai/mturk/core/dev/server_utils.py

```diff
@@ -447,18 +450,18 @@ def setup_heroku_server(task_name, task_files_to_copy=None,
     os.chdir(parent_dir)
```

Comment:
same question here

Reply:
I can actually drop this line and any other `os.chdir`'s as I'm using absolute paths for everything now, but I'll do that in the dev branch cleanup coming soon.